keps/sig-api-machinery/4355-coordinated-leader-election/README.md

```diff
@@ -849,15 +850,22 @@ in back-to-back releases.
 
 #### Beta
 
-- e2e tests
+- e2e tests for coordinated leader election on various scenarios
+  + single leasecandidate
+  + multiple leasecandidates
+  + lease is preempted when another more suitable candidate is found
```

Comment:
- old components that don't know about coordination, mixed with those that do
- downgrade to components that don't know about coordination (not sure about the overlap with the "disablement" test below).
- custom strategy controller.

Reply:
added